torch/linalg/__init__.py

```diff
@@ -483,6 +483,53 @@
     >>> tensor([ 11.6734+0.j, 105.1037+0.j,  10.1978+0.j])
 """)
 
+solve = _add_docstr(_linalg.linalg_solve, r"""
+linalg.solve(input, other, *, out=None) -> Tensor
+
+Computes the solution ``x`` to the matrix equation ``matmul(input, x) = other``
+with a square matrix, or batches of such matrices, :attr:`input` and one or more right-hand side vectors :attr:`other`.
+If :attr:`input` is batched and :attr:`other` is not, then :attr:`other` is broadcasted
```

Comment:
"is broadcasted" -> "is broadcast"

Reply:
If the array is _broadcastable_, then it is _broadcast_ across the larger array. We do _broadcasting_ of the smaller array and the result is the _broadcasted_ array.

For A, b in Ax=b. If `A` is batched and `b` isn't -> `b` is _broadcasted_ to have the same batch dimensions `A`. `x` has the same shape as the (possibly _broadcasted_) `b`.

See https://numpy.org/doc/stable/reference/generated/numpy.broadcast.html#numpy.broadcast
> `index` current index in **broadcasted** result
> `shape` Shape of **broadcasted** result.
> `size` Total size of **broadcasted** result.

Maybe we shouldn't use this term here anyway? Since we don't really do the broadcasting but create internally full copies with the compatible shape. To the user, it looks like broadcasting though.

Reply:
I appreciate your point and if you prefer "broadcasted" that's OK.

As best I can tell "broadcasted" seems to be a less common but acceptable past tense for the word "broadcast." Here's an article arguing for "broadcast" vs. "broadcasted":
https://www.merriam-webster.com/words-at-play/is-it-broadcast-or-broadcasted-forecast-or-forecasted-usage. I would prefer we follow the AP Stylebook's recommendation and avoid it in favor of the more common past tense for "broadcast."

Reply:
Thank you for the link! I've changed it to "broadcast".